spec/ascii-art-spec.coffee

```diff
@@ -7,15 +6,29 @@ AsciiArt = require '../lib/ascii-art'
 # or `fdescribe`). Remove the `f` to unfocus the block.
 
 describe "AsciiArt", ->
-  promise = null
+  [workspaceElement, activationPromise] = []
+
   beforeEach ->
-    atom.workspaceView = new WorkspaceView()
-    atom.workspace = atom.workspaceView.model
-    promise = atom.packages.activatePackage('ascii-art')
+    workspaceElement = atom.views.getView(atom.workspace)
+    activationPromise = atom.packages.activatePackage('ascii-art')
+
     waitsForPromise ->
       atom.workspace.open()
 
   it "converts", ->
-    atom.workspaceView.trigger 'ascii-art:convert'
+    editor = atom.workspace.getActiveTextEditor()
+    editor.insertText("yo")
+    editor.selectAll()
+    changeHandler = jasmine.createSpy('changeHandler')
+    editor.onDidChange(changeHandler)
+
+    atom.commands.dispatch workspaceElement, 'ascii-art:convert'
+
     waitsForPromise ->
-      promise
+      activationPromise
+
+    waitsFor ->
+      changeHandler.callCount > 0
+
+    runs ->
+      expect(editor.getText()).toEqual "\n                 \n                 \n __  __    ___   \n/\\ \\/\\ \\  / __`\\ \n\\ \\ \\_\\ \\/\\ \\L\\ \\\n \\/`____ \\ \\____/\n  `/___/> \\/___/ \n     /\\___/      \n     \\/__/       \n"
```

Comment:
I'd like to see this broken across multiple lines in the source with a `"""`-style string. Did you try that and run into trouble? All on one line like this it's a bit tough to read. Considering this is an example I think it would be nice to have it be as clear as possible.